proversity_reports_script/report_backend/completion_report.py

```diff
@@ -8,17 +8,18 @@
 
 import boto3
 
-from proversity_reports_script.report_backend.base import \
-    AbstractBaseReportBackend
+from proversity_reports_script.google_apis.sheets_api import update_sheets_data
+from proversity_reports_script.report_backend.base import AbstractBaseReportBackend
 
 
 class CompletionReportBackend(AbstractBaseReportBackend):
     """
     Backend for Completion report.
     """
 
-    def __init__(self, **kwargs):
-        super(CompletionReportBackend, self).__init__(**kwargs)
+    def __init__(self, *args):
+        super(CompletionReportBackend, self).__init__(*args)
+
```

Comment:
remove this empty space

Reply:
Addressed, but i think that two blank lines is better for code reading, although pep8 says the opposite.